trade-finance-manager-ui/templates/case/facility/_macros/amendment-details.njk

```diff
@@ -0,0 +1,141 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{%- from "govuk/components/tag/macro.njk" import govukTag -%}
+
+{% macro render(params) %}
+  {% set amendmentVersion = params.amendmentVersion %}
+  {% set showContinueAmendmentButton = params.showContinueAmendmentButton %}
+  {% set allAmendments = params.allAmendments %}
+
+  {% for amendment in allAmendments %}
+    {% if amendment.submittedByPim === true %}
+      <div class="govuk-grid-row govuk-!-margin-top-53 govuk-!-padding-top-6 amendment-details" data-cy="amendment--details-{{amendment.version}}">
+        <div class="govuk-grid-column-full">
+          <h2 class="govuk-heading-l" data-cy="amendment--heading-version-{{amendment.version}}"> Amendment {{amendment.version}} </h2>
+          <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-quarter">
+              <h4 class="govuk-heading-s govuk-body-s govuk-!-font-weight-bold">Date amendment is effective from</h4>
+            </div>
+
+            <div class="govuk-grid-column-three-quarters">
+              <p class="govuk-body-s" data-cy="amendment--details-{{amendment.version}}-effective-date">{{ amendment.effectiveDate | dashIfEmpty}}</p>
+            </div>
+          </div>
+
+          <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-quarter">
+              <h4 class="govuk-heading-s govuk-body-s govuk-!-font-weight-bold">Bank requested change</h4>
+            </div>
+
+            <div class="govuk-grid-column-three-quarters">
+              <p class="govuk-body-s" data-cy="amendment--details-{{amendment.version}}-request-date">{{ amendment.requestDate | dashIfEmpty}}</p>
+            </div>
+          </div>
+
+          <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-quarter">
+              <h4 class="govuk-heading-s govuk-body-s govuk-!-font-weight-bold">UKEF approval required</h4>
+            </div>
+
+            <div class="govuk-grid-column-three-quarters">
+              <p class="govuk-body-s" data-cy="amendment--details-{{amendment.version}}-require-approval">{{ amendment.requireUkefApproval | dashIfEmpty}}</p>
```

Comment:
XSS Filter please `safe`.

Reply:
I'm not sure that's needed here. the `requireUkefApproval` is created inside [here](https://github.com/UK-Export-Finance/dtfs2/blob/feature/amendment-details/trade-finance-manager-ui/server/controllers/case/index.js#L187)

the value can only be `Yes` or `No`